app/components/forms/wizard/basic-details-step.js

```diff
@@ -269,6 +283,44 @@ export default Component.extend(FormMixin, EventWizardMixin, {
             }
           ]
         },
+        ticketMinPrice: {
+          identifier : 'min_price',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Minimum price for donation tickets required')
+            },
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Invalid number')
+            },
+            {
+              type   : 'checkZero',
+              prompt : this.l10n.t('Ticket Price should be greater than zero')
```

Comment:
For this, you may just use a check for type `integer[1..n]`, No need for two tests.